src/uikit.cs

```diff
@@ -25415,10 +25415,15 @@ interface UIDeferredMenuElement {
 		[Export ("elementWithProvider:")]
 		UIDeferredMenuElement Create (UIDeferredMenuElementProviderHandler elementProvider);
 
-		[iOS (15, 0), MacCatalyst (15, 0)]
+		[iOS (15, 0), MacCatalyst (15, 0), TV (15, 0)]
 		[Static]
 		[Export ("elementWithUncachedProvider:")]
+#if XAMCORE_5_0
 		UIDeferredMenuElement CreateUncached (UIDeferredMenuElementProviderHandler elementProvider);
+#else
+		[Internal]
+		UIDeferredMenuElement _CreateUncached (UIDeferredMenuElementProviderHandler elementProvider);
+#endif
```

Comment:
Actually I don't think you need the compat code, just adding the "[TV (15, 0)]" attribute will only change the generated attributes on the member, and that's not a breaking change. The compat code will make the API work on older tvOS versions (when previously it didn't).

Reply:
But previously would throw an exception when used, correct? We wonder what is better. We did not get anyone complaining so we might be ok

Reply:
I think it's OK to keep doing what we did (throw an exception). It's easy enough for developers to fix, and with the updated attribute they'll even get a warning saying what the problem is.